__tests__/unit/Directions/DifferentBuildingDirections.test.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { DifferentFloorDirections } from "../../../components/IndoorDirections/TypesOfDirections/DifferentFloorDirections";
+import { HallXCoordinates } from "../../../constants/HallXCoordinates";
+import { getFloorNumber, changeClassName, dijkstra } from "../../../components/IndoorDirections/Dijkstra/DijkstraAlgorithm";
+import { ClassGraph } from "../../../constants/ClassGraph";
+
+describe("DifferentBuildingDirections component", () => {
+    test("renders correctly", () => {
+        const tree = renderer.create(<DifferentFloorDirections floor={8} rooms={HallXCoordinates()} from={"H835"} to={"MB3.125"}/>).toJSON();
```

Comment:
Why are you using the component _DifferentFloorDirections_ to test different buildings?  If _DifferentFloorDirections_ can handle different buildings configuration, maybe it should have a more generic name like _IndoorDirections_ .

Reply:
Because the snapshots of all three paths vary

Reply:
I think it comes down to how you structured your indoor map functionality.